config/software/ruby.rb

```diff
@@ -28,15 +28,15 @@
 # - Note: When updating major version (i.e. 2.2 -> 2.3), gem_dir_version must be
 #   updated to 2.X.0 in td.rb, td-agent-cleanup.rb, td-agent-files.rb, and
 #   td-agent-ui.rb.
-default_version "2.4.3"
+default_version "2.5.3"
 
 fips_enabled = (project.overrides[:fips] && project.overrides[:fips][:enabled]) || false
 
 dependency "patch" if solaris_10?
 dependency "ncurses" unless windows? || version.satisfies?(">= 2.1")
 dependency "zlib"
 dependency "openssl"
-dependency "libedit"
+dependency "libedit" unless windows?
```

Comment:
Did this break anything in Windows? If so, can we add a comment for future reference?

Reply:
Honestly, I just grabbed the changes from the upstream repo.

Reply:
Oh I see. Cool then.